scripts/dotnet-cli-build/PublishTargets.cs

```diff
@@ -118,6 +116,41 @@ public static BuildTargetResult PublishDebFileToDebianRepo(BuildTargetContext c)
             return c.Success();
         }
 
+        [Target]
+        [Environment("DOCKER_HUB_REPO")]
```

Comment:
You must restrict this target to run only on on Ubuntu or whichever platform for which you are trying to build the docker image.